tests/helper/helper_dev.go

```diff
@@ -184,8 +184,8 @@ func (o DevSession) CheckNotSynced(timeout time.Duration) {
 // RunDevMode runs a dev session and executes the `inside` code when the dev mode is completely started
 // The inside handler is passed the internal session pointer, the contents of the standard and error outputs,
 // and a slice of strings - ports - giving the redirections in the form localhost:<port_number> to access ports opened by component
-func RunDevMode(inside func(session *gexec.Session, outContents []byte, errContents []byte, ports map[string]string)) error {
-	session, outContents, errContents, urls, err := StartDevMode()
+func RunDevMode(inside func(session *gexec.Session, outContents []byte, errContents []byte, ports map[string]string), additionalOpts ...string) error {
```

Comment:
I find it more readable if the additionalOpts is the first parameter, before the lambda
- you can read 'RunDevMode(--run-command, ...)  directly
- you don't have to scroll the entire lambda code to know the other parameters

Reply:
I didn't want to touch the other places where this function was being called, but okay, I'll change that as well.